lib/rules/disallow-multiple-spaces.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Disallows multiple spaces between identifiers, keywords, and any other token
+ *
+ * Type: `Boolean`
+ *
+ * Values: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "disallowMultipleSpaces": true
+ * ```
+ *
+ * ##### Valid
+ * ```js
+ * var x = "hello";
+ * function y() {}
+ * ```
+ *
+ * ##### Invalid
+ * ```js
+ * var x  = "hello";
+ * function y() {}
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+
+    configure: function(disallowMultipleSpaces) {
+        assert(
+            typeof disallowMultipleSpaces === 'boolean',
+            'disallowMultipleSpaces option requires boolean value'
+        );
+        assert(
+            disallowMultipleSpaces === true,
+            'disallowMultipleSpaces option requires true value or should be removed'
+        );
+    },
+
+    getOptionName: function() {
+        return 'disallowMultipleSpaces';
+    },
+
+    check: function(file, errors) {
+        file.getTokens().forEach(function(token, index, tokens) {
```

Comment:
Ah, didn't realize we were missing those tokens. Good catch. Do you think there would be a reuse value in an `iterateTokens` with options for including comment tokens?